servers/audio_server.cpp

```diff
@@ -515,6 +515,15 @@ String AudioServer::get_bus_name(int p_bus) const {
 	return buses[p_bus]->name;
 }
 
+int AudioServer::get_bus_index(const StringName &p_bus_name) const {
+
+	if (bus_map.has(p_bus_name)) {
+		Bus *bus = bus_map[p_bus_name];
+		return buses.find(bus);
```

Comment:
this is probably redundant, just do a for() int the buses to find the right one..